morphir/sdk/core/src/morphir/sdk/Basics.scala

```diff
@@ -108,6 +108,10 @@ object Basics {
     if (a < min) min
     else if (a > max) max
     else a
+  @inline def logBase(base: Float, number: Float): Float =
+    divide(scala.math.log(number))(scala.math.log(base))
+  @inline def e(a: Float): Float =
+    scala.math.exp(a)
```

Comment:
This doesn't look right. `e` just returns a number in Elm: https://package.elm-lang.org/packages/elm/core/latest/Basics#e

Reply:
`Math.E` looks like the right thing to use here: https://docs.oracle.com/javase/8/docs/api/java/lang/Math.html#E

Reply:
Okay @AttilaMihaly, I will make the necessary changes.

Reply:
@AttilaMihaly Just I mistook e() for exp() math function